### PR TITLE
fix some test failed

### PR DESCRIPTION
**Pull request by stewart-yu**
**What this PR does / why we need it**:
fix test case failed.
(1) seems that `--adress` and `--port `deleted in flag, we can't not convert it, so delete it .
(2) now that add some flag, we should test it.
(3) we change address to '0.0.0.0' for to match value in `NewDefaultControllerManagerComponentConfig()`, because the test case can't run `InsecureServingOptions.ApplyTo`.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/cloud-controller-manager/app/options/options_test.go [hunk 1]
```diff
@@ -35,7 +38,6 @@ func TestAddFlags(t *testing.T) {
 	s.AddFlags(f)
 
 	args := []string{
-		"--address=192.168.4.10",
```
Comment (sttts) [inline cmd/cloud-controller-manager/app/options/options_test.go]: why are those removed?

Comment (stewart-yu) [reply cmd/cloud-controller-manager/app/options/options_test.go]: `address` flag removed, so  we no need to rewrite `address`

File: cmd/cloud-controller-manager/app/options/options_test.go [hunk 2]
```diff
@@ -56,88 +58,104 @@ func TestAddFlags(t *testing.T) {
...
+		"--cert-dir=/a/b/c",
+		"--bind-address=192.168.4.21",
+		"--secure-port=10001",
 	}
 	f.Parse(args)
 
+	componentConfig := componentconfig.KubeControllerManagerConfiguration{
+		CloudProvider:                                   "gce",
+		CloudConfigFile:                                 "/cloud-config",
+		Port:                                            ports.InsecureCloudControllerManagerPort,
```
Comment (sttts) [inline cmd/cloud-controller-manager/app/options/options_test.go]: this should be the number, not the constant. This test also makes sure that the actual config is not changed. Referencing constants is elegant, but against the purpose here.

File: cmd/cloud-controller-manager/app/options/options_test.go (hunk 1 above)
Comment (stewart-yu) [reply cmd/cloud-controller-manager/app/options/options_test.go]: The Address is set in `InsecureServingOptions.ApplyTo`, so removed here as kube-apiserever options test.

File: cmd/cloud-controller-manager/app/options/options_test.go (hunk 2 above)
Comment (stewart-yu) [reply cmd/cloud-controller-manager/app/options/options_test.go]: ok, will fix now

**Comment by stewart-yu** [comment]
seem fixed,  close
